proposals/36-usernet.md

```diff
@@ -0,0 +1,205 @@
+
+#### Background & intent
+_Describe the desired state of the world after this project? Why does that matter?_
+
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve. Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives. 
+-->
+
+##### User experience (UX)
+
+Imagine I'm a potential user or developer that comes across Filecoin on the interwebs.
+I think "this sounds cool, let's give it a try".
+This sends me down a rabbit hole:
+
+- I need to purchase FIL in an exchange.
+- For this I need to have an account at an exchange.
+- Assuming I don't, I will need to go through KYC and surrender a lot of personal data and documents. Then I need to wait to get my account approved.
+- Then I need to send funds on my exchange wallet. If I don't own any crypto, I'll have to wire fiat, which won't arrive immediately. Depending on my banking situation, this operation may not be free of charge.
+- Wait until the funds arrive.
+- Ok, good! The funds are there. I might be in a position to actually buy the token now.
+- Let's figure out how this exchange works... the order book... what is a market order? What is spot? What is margin? Whaaaat?
+- Yes! I bought 10 USD worth of FIL. Now what do I do?
+- Ok, I need to download a Filecoin client, create a wallet, and withdraw my FIL to it.
+- Nice, now I have FIL. (Oops, did I realise that I now may have new tax reporting obligations, just because I wanted to try out this thing?)
+- (Insert deal making process and associated pain, see [Project F3 pitch](https://github.com/protocol/web3-dev-team/pull/12)).
+
+Compare with Dropbox (personal data) / S3 (API-based cloud storage provider):
+
+1. Sign up for free.
+2. Try out the thing.
+
+##### Developer experience (DX)
+
+Developers building on Filecoin have three options:
+
+1. Test on testnet: now defunct.
+2. Test on mainnet: requires real money, and iteration is slow due to sealing times, paying for unnecessary guarantees (e.g. 180d deal duration; when testing or developing, you don't need that much).
+3. Set up their own local devnet: complex, heavy, requires infrastructure.
+
+Ideally, we'd provide a network that reproduces the conditions of mainnet (including faults), that is hosted, for developers to iterate rapidly and exercise the various scenarios that can occur in the Filecoin integration (happy paths, functional errors, faults, expiries, etc.)
+
+##### A dedicated network
+
+This proposal aims to introduce a new Filecoin network: a **permissioned, stable testnet** that is operated by Protocol Labs and other trusted partners, with zero-cost deals, some form of quick sealing, and short deal duration, so that new users can get started immediately without having to buy FIL, spend a dime, or have to suffer whatever the current market dynamics are (miner market, gas market, etc).
```

Comment:
* zero-cost deals => as long as there's a faucet, deals can be zero-cost.
* fast sealing => not really; PoRep performance depends on hardware, and is geared towards adversarial, trustless networks. There are two ways to make PoRep faster:
    - (a) reduce sector size to e.g. 8MiB (which entirely changes the characteristics of the network and the use cases it can support, and is no longer a replica of mainnet), or 
    - (b) more powerful hardware (which is non-desirable for a testnet).
* shorter deal duration => I'd like to support from minutes to hours. Standard PoRep won't work with that range.